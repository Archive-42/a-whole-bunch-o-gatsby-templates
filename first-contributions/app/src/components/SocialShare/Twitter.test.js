import React from 'react';
import renderer from 'react-test-renderer';
import TwitterCard from './Twitter';

it('renders a project card', () => {
  const tree = renderer.create(
    <TwitterCard/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

