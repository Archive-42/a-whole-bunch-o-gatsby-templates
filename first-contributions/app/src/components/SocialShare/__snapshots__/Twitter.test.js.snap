// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a project card 1`] = `
<a
  className="icon-card twitter"
  href="https://twitter.com/intent/tweet?text=Yay%21%20I%20just%20made%20my%20first%20open%20source%20contribution%20with%20@1stcontribution.%20You%20can%20too%20at%20https%3A//goo.gl/66Axwe%0A&hashtags=Hacktoberfest,OpenSource"
  target="_blank"
>
  <svg
    height="40px"
    style={
      Object {
        "display": "block",
        "margin": "0 auto",
      }
    }
    viewBox="0 0 22 22"
    width="40px"
  >
    <path
      d="M22 4.864c-0.808 0.361-1.68 0.602-2.591 0.709 0.932-0.559 1.646-1.444 1.985-2.496-0.872 0.516-1.839 0.894-2.866 1.096-0.825-0.877-1.998-1.422-3.296-1.422-2.492 0-4.512 2.020-4.512 4.512 0 0.352 0.039 0.696 0.116 1.027-3.751-0.189-7.077-1.985-9.303-4.718-0.387 0.666-0.61 1.444-0.61 2.269 0 1.564 0.795 2.948 2.007 3.755-0.739-0.021-1.435-0.228-2.045-0.563 0 0.017 0 0.039 0 0.056 0 2.187 1.555 4.013 3.622 4.426-0.378 0.103-0.778 0.159-1.19 0.159-0.292 0-0.571-0.030-0.851-0.082 0.576 1.792 2.243 3.098 4.215 3.137-1.547 1.212-3.489 1.934-5.607 1.934-0.365 0-0.722-0.021-1.079-0.064 2.002 1.285 4.374 2.028 6.922 2.028 8.302 0 12.843-6.879 12.843-12.843 0-0.198-0.004-0.391-0.013-0.584 0.881-0.632 1.646-1.427 2.252-2.333z"
      style={
        Object {
          "fill": "rgb(255, 255, 255)",
        }
      }
    />
  </svg>
</a>
`;
